translations/en.i18n.json

```diff
@@ -0,0 +1,10 @@
+{
+  "en": {
```

Comment:
@lianbenjamin If we have the texts somewhere copy them from there, the rest will need to be translated